packages/edit-site/src/components/global-styles/screen-link-color.js

```diff
@@ -15,6 +15,11 @@ import {
 	useColorsPerOrigin,
 } from './hooks';
 
+function ucFirst( str ) {
+	if ( ! str ) return str;
+	return str[ 0 ].toUpperCase() + str.slice( 1 );
+}
+
```

Comment:
I think we're trying to ditch Lodash so I thought this was simple enough as a helper.

We could get clever and use destructuring but this is a method I had stored in my personal knowledge base.